src/framework/components/element/text-element.js

```diff
@@ -900,9 +911,14 @@ class TextElement {
                     _xMinusTrailingWhitespace = _x;
                 }
 
-                var isWordBoundary = WORD_BOUNDARY_CHAR.test(char);
-                var isCJK = CJK_CHAR.test(char);
-                if (isWordBoundary || isCJK) { // char is space, tab, or dash OR CJK
+                // char is space, tab, or dash
+                const isWordBoundary = WORD_BOUNDARY_CHAR.test(char);
+                // char is CJK character boundary
+                const isCJKBoundary = CJK_CHAR.test(char) && (((nextchar !== null) && (WORD_BOUNDARY_CHAR.test(nextchar)) || ALPHANUMERIC_CHAR.test(nextchar)));
```

Comment:
I think the brackets should be like so:
```suggestion
                const isCJKBoundary = CJK_CHAR.test(char) && ((nextchar !== null) && (WORD_BOUNDARY_CHAR.test(nextchar) || ALPHANUMERIC_CHAR.test(nextchar)));
```

Reply:
thanks :)